doc/ArchInstall/arch-chroot_install.sh

```diff
@@ -184,6 +184,13 @@ systemctl enable systemd-resolved.service
 systemctl enable wpa_supplicant
 systemctl enable wpa_supplicant@${WIFI_INTERFACE}
 
+#####################
+# SERIAL PORT RULES #
+#####################
+cat << EOF > /etc/udev/rules.d/10-nusense.rules
+KERNEL=="ttyACM*", SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ATTRS{manufacturer}=="STMicroelectronics", ATTRS{idProduct}=="5740", ATTRS{idVendor}=="0483", SYMLINK+="nusense", GROUP="uucp", MODE="0666"
+EOF
+
```

Comment:
Remove this. You are adding a config file to `SystemConfiguration`, that is a much better option than the OS install script